src/components/containerOurWorks/worksPlaceholders/WorksPlaceholders.js

```diff
@@ -3,14 +3,29 @@ import "./WorksPlaceholders.css";
 import WorksPlaceholder from "./worksPlaceholder/WorksPlaceholder";
 
 class WorksPlaceholders extends Component {
+  state = {
+    list: undefined
+  };
+
+  getImg = () => {
+    fetch("https://5cb6f62aa3763800149fd07a.mockapi.io/api/works")
+      .then(response => response.json())
+      .then(data => this.setState({ list: data }))
+      .catch(error => console.error(error));
+  };
+
+  componentDidMount() {
+    this.getImg();
+  }
   render() {
-    return (
-      <div className="our-works-placeholders">
-        <WorksPlaceholder />
-        <WorksPlaceholder />
-        <WorksPlaceholder />
-      </div>
-    );
+    let listComponents;
+    const listResp = this.state.list;
```

Comment:
this could be written as: 

```suggestion
    const { list } = this.state;
```

Read more about this in the destructive assignment article here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment